pkg/updater/download_test.go

```diff
@@ -23,44 +23,155 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+	"github.com/ulikunitz/xz"
 )
 
 const (
 	testAgentFileName        = "agent"
 	testNestedAgentFileName  = "nested/agent2"
 	testLargeFileName        = "large"
-	testLargeFileSize        = 1024 * 1024 * 20 // 20MB
+	testLargeFileSize        = 1024 * 1024 * 5 // 5MB
```

Comment:
Reduced to avoid tests timeouts when building the archive. The tests are effectively the same.